src/payment/strategies/paypal-commerce/paypal-commerce-payment-initialize-options.ts

```diff
@@ -89,6 +90,7 @@ export interface PaypalCommercePaymentInitializeOptions {
      * The CSS selector of a container where the alternative payment methods fields widget should be inserted into.
      * It's necessary to specify this parameter when using Alternative Payment Methods.
```

Comment:
Is there a better way to surface this, as @icatalina pointed previously that anything with `?:` is not required 🤷‍♂️

Reply:
As i see container is required and don't have ? (optional sign)

Reply:
Hey @andriiVitvitskyi1990, you are right, `container` is already flagged as mandatory. The goal is to use a consistent way of flagging this across the codebase, as opposed to adding custom notes to every mandatory field. 

If we feel the documentation is not clear enough using `?:` and the `Optional` label (https://github.com/bigcommerce/checkout-sdk-js/blob/master/docs/interfaces/paypalcommercepaymentinitializeoptions.md#optional-apmfieldscontainer) we can look into ways of improving it by tweaking the markdown generator.

Reply:
@icatalina  We addressed this offline, @andriiVitvitskyi1990 thought the comment was for `container` field 😄 And then adjusted the text accordingly.